packages/styles/src/variables/_elevation.scss

```diff
@@ -1,27 +1,17 @@
 @use 'sass:map';
 
-// Values are kept in PX for easier copy-ability from figma and better overview
-// rems would provide little value added
-$elevation-1:
-  0.25px 1px 1px rgba(0, 0, 0, 0.1),
-  1px 2px 2.5px 1px rgba(0, 0, 0, 0.1);
-$elevation-2:
-  0.25px 1.5px 2px rgba(0, 0, 0, 0.1),
-  2px 2px 5px 2px rgba(0, 0, 0, 0.11);
-$elevation-3:
-  0.5px 3px 4px rgba(0, 0, 0, 0.1),
-  2px 4px 10px 4px rgba(0, 0, 0, 0.12);
-$elevation-4:
-  0.75px 4.5px 6px rgba(0, 0, 0, 0.1),
-  2px 6px 15px 6px rgba(0, 0, 0, 0.13);
-$elevation-5:
-  1px 6px 8px rgba(0, 0, 0, 0.1),
-  2px 8px 20px 8px rgba(0, 0, 0, 0.14);
+$elevation-none: none;
+$elevation-1: var(--post-core-elevation-1);
```

Comment:
Core tokens cannot be used in the styles, you need to request a specific set of utility tokens here.